runtime/src/bank.rs

```diff
@@ -795,6 +795,7 @@ impl Bank {
             self.collect_fees();
             self.distribute_rent();
             self.update_slot_history();
+            self.run_incinerator();
```

Comment:
Care to add some timing `if`? Otherwise, we're vulnerable to simple dos within the time window where supermajority validators are mixed with old (just simple and bogus account) and new (it'll immediately disappear bunring capitalization).

Reply:
yeah, yuck: I'll do this only on v1.1 and v1.0 though - https://github.com/solana-labs/solana/issues/9821